pytest_splunk_addon/docker_class.py

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright 2021 Splunk Inc.
```

Comment:
@artemrys I have added this copyright license. Hope that is fine.

Reply:
Actually I am not sure - I'll ask our legal person if that's okay.

Please put it as an exception for now to `.licenserc.yaml`.

Reply:
Will do

Reply:
Done